kie-server-parent/kie-server-controller/kie-server-controller-websocket/src/main/java/org/kie/server/controller/websocket/client/WebSocketKieServerClient.java

```diff
@@ -92,6 +93,14 @@ public WebSocketKieServerClient(String url) {
         throw new UnsupportedOperationException("Not supported for Web Socket implementation");
     }
 
+    private List<?> safeList(List<?> list) {
+        if (list == null) {
+
+            return new ArrayList<Object>();
+        }
+        return new ArrayList<Object>(list);
+    }
+
```

Comment:
Maybe replace this function by something like this  Optional.ofNullable(list).orElse(Collections.emptyList());

Reply:
ok

Reply:
```suggestion
return list == null ? emptyList() : list
```